jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/EnvUtil.java

```diff
@@ -446,12 +456,33 @@ public static boolean isWindows() {
         return System.getProperty("os.name").toLowerCase().contains("windows");
     }
 
+    /**
+     * Return a {@link File} pointing to the user's home directory.
+     * @return the user home directory.
+     */
+    public static File getUserHome() {
```

Comment:
Is it possible to replace this with https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SystemProperties.html#getUserHome-- ?

Reply:
This method is more complete.